src/pilot/discover_nodes/discover_nodes/dracclient/client.py

```diff
@@ -93,7 +93,7 @@ def list_idrac_settings(self):
         :raises: DRACOperationFailed on error reported back by the DRAC
                  interface
         """
-        return self._idrac_cfg.list_idrac_settings()
+        return self._idrac_con.list_idrac_settings()
```

Comment:
Please see my comment above about the `_idrac_con` data member. If that's removed, this method must be removed, too.

Reply:
Done, Please review